nuget/updater/main.ps1

```diff
@@ -0,0 +1,86 @@
+Set-StrictMode -version 2.0
+$ErrorActionPreference = "Stop"
+
+$jobString = Get-Content -Path $env:DEPENDABOT_JOB_PATH
+$job = (ConvertFrom-Json -InputObject $jobString).job
+
+function Get-Files {
+    Write-Host "Job: $($job | ConvertTo-Json)"
+    $sourceRepo = $job.source.repo
+    # TODO: handle other values from $job.source.provider
+    $url = "https://github.com/$sourceRepo"
+    $branchArgs = ""
+    if ("branch" -in $job.source.Psobject.Properties.Name) {
+        $branchArgs = "--branch $($job.source.branch)"
+    }
+
+    Invoke-Expression "git clone --recurse-submodule $branchArgs $url $env:DEPENDABOT_REPO_CONTENTS_PATH"
+    if ("commit" -in $job.source.PSobject.Properties.Name) {
+        git fetch origin
+        git checkout $job.source.commit
+    }
+}
+
+function Install-Sdks([string] $directory) {
+    $installedSdks = dotnet --list-sdks | ForEach-Object { $_.Split(' ')[0] }
+    if ($installedSdks.GetType().Name -eq "String") {
+        # if only a single value was returned (expected in the container), then force it to an array
+        $installedSdks = @($installedSdks)
+    }
+    Write-Host "Currently installed SDKs: $installedSdks"
+    $rootDir = Convert-Path $env:DEPENDABOT_REPO_CONTENTS_PATH
+    $candidateDir = Convert-Path "$rootDir/$directory"
+    while ($true) {
+        $globalJsonPath = Join-Path $candidateDir "global.json"
+        if (Test-Path $globalJsonPath) {
+            $globalJson = Get-Content $globalJsonPath | ConvertFrom-Json
+            $sdkVersion = $globalJson.sdk.version
```

Comment:
Given the way PowerShell invokes commands, the `$sdkVersion` variable will always be passed as a parameter, even if that part of `global.json` looked like ` && 1; rm -rf /` but even still, this would be the user sabotaging their own code and the container this runs in has no access to secrets, so worst case is they don't get updates.